packages/wrangler/src/__tests__/deploy.test.ts

```
@@ -8769,24 +8769,10 @@ addEventListener('fetch', event => {});`
 							},
 						} as unknown as ChildProcess;
 					})
-					// 4. docker tag
```

we don't need to tag anymore because we just build the image with the intended tag. afaict, this was already the case for deploy, and this was probably a no-op or a swallowed error.